data/styles/clear/include/Icons.mapcss

```diff
@@ -910,28 +914,22 @@ node|z17-[amenity=ferry_terminal]
 
 /* 3.6 Airport */
 
-/* Manually define priority to avoid city names displacement */
 area|z7-9[aeroway=aerodrome][aerodrome=international],
 node|z7-9[aeroway=aerodrome][aerodrome=international],
-{ icon-image: airport-s.svg; -x-me-icon-priority: 10000 }
+{icon-image: airport-s.svg;}
```

Comment:
Hm, I added it _recently_ and there were reasons.
Can you check that airports don't displace cities now on z7? Let say Warsaw and Frankfurt-Maine?

Reply:
As I wrote before - this fix doesn't change any relative priority.
International airports are not displacing cities and e.g. states, because airports are z7- and cities are z4-.
Int. airports are displacing towns, because towns are z8-.
Regardless of the priority value set in the styles.

Do you have a real case when its not working like this?

Reply:
Hm, show me Warsaw, Frankfurt-Maine, Gdansk screenshots on z7 with this change and regenerated styles.

Reply:
> Hm, show me Warsaw, Frankfurt-Maine, Gdansk screenshots on z7 with this change and regenerated styles.

I triple-checked, airports are not displacing these cities. There is no change on the screenshots. Just check with your own eyes if you don't believe :)

Reply:
Ok then :) Approve.